src/Compilers/CSharp/Test/Semantic/Semantics/ValueTupleTests.cs

```diff
@@ -0,0 +1,129 @@
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3__Item3));
+
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4__Item1));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4__Item2));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4__Item3));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4__Item4));
+
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5__Item1));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5__Item2));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5__Item3));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5__Item4));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5__Item5));
+
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6__Item1));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6__Item2));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6__Item3));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6__Item4));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6__Item6));
+
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7__Item1));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7__Item2));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7__Item3));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7__Item4));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7__Item6));
+            Assert.NotNull(comp.GetWellKnownTypeMember(WellKnownMember.System_Runtime_CompilerServices_ValueTuple_T1_T2_T3_T4_T5_T6_T7__Item7));
```

Comment:
Consider asserting result of ToTestDisplayString rather than simply checking for not null.